source/contracts/reporting/ReportingToken.sol

```diff
@@ -115,7 +117,9 @@ contract ReportingToken is DelegationTarget, Typed, Initializable, VariableSuppl
         if (_reporterReputationShare != 0) {
             _reputationToken.transfer(msg.sender, _reporterReputationShare);
         }
-        _reportingWindow.collectReportingFees(msg.sender, _attotokens);
+        if (!forgoFees) {
+            _reportingWindow.collectReportingFees(msg.sender, _attotokens);
+        }
```

Comment:
If the user is forgoing fees, we will need to deduct their reporting tokens from the total here, so other people get their share of fees and the fees don't end up sitting unclaimed.  This is over in `collectReportingFees` function, we basically need to do the same thing just without paying out.
```
totalWinningReportingTokens = totalWinningReportingTokens.sub(_attoReportingTokens);
```

Reply:
Also recommend adding an explicit test for this, since its a subtlety that is easy to miss/break in the future.

Reply:
Good catch, and will do